apps/api/api-cf.yml

```diff
@@ -3,6 +3,9 @@ Parameters:
   JobsTable:
     Type: String
 
+  UserTable:
```

Comment:
We have a mix of singular/plural.  Should be user/job everywhere or users/jobs everywhere, I don't have a strong opinion which.